apps/web/build/generateScssVariables.ts

```diff
@@ -0,0 +1,70 @@
+import fs from 'node:fs';
+import path from 'node:path';
+import { getTailwindColorsOklch } from '../utils/tailwindHelper';
+import type { TailwindColors } from '~/composables/useConfigurationDrawer';
+import { rgb } from 'culori';
+import { formatRgb } from 'culori/fn';
+
+const primaryColor = process.env.PRIMARY || '#0c7992';
+const secondaryColor = process.env.SECONDARY || '#008ebd';
+
+export const oklchToRgb = (oklch: string) => {
+  const rgbColor = rgb(oklch) ?? { mode: 'rgb', r: 0, g: 0, b: 0 };
+  const rgbColorFormatted = formatRgb(rgbColor);
+  const rgbColorString = rgbColorFormatted.slice(rgbColorFormatted.indexOf('(') + 1, rgbColorFormatted.indexOf(')'));
+
+  return rgbColorString.replaceAll(',', '');
+};
+
+export const convertOklchToRgb = (tailwindColors: TailwindColors) => {
```

Comment:
```suggestion
const convertOklchToRgb = (tailwindColors: TailwindColors) => {
```
**todo**: Not used elsewhere.